decidim-comments/app/packs/src/decidim/comments/comments.component.js

```diff
@@ -377,6 +377,7 @@ export default class CommentsComponent {
 
     $opinionButtons.removeClass("is-active").attr("aria-pressed", "false");
     $btn.addClass("is-active").attr("aria-pressed", "true");
+    $btn.css("fontWeight", "bold");
```

Comment:
we could add this via CSS, to avoid the bug in which you select one opinion, then change it... :
![image](https://github.com/user-attachments/assets/12e1c306-3f9c-48e4-bd82-917ecbbd04a3)

decidim-comments/app/packs/stylesheets/comments.scss, line 405
```suggestion
```

Reply:
Hello @alecslupu , thanks for your review, I have applied your suggestions !